cache/clock_cache.h

```diff
@@ -393,13 +675,16 @@ class ALIGN_AS(CACHE_LINE_SIZE) ClockCacheShard final : public CacheShard {
 
  private:
   friend class ClockCache;
-  void ClockRemove(ClockHandle* e);
-  void ClockInsert(ClockHandle* e);
+
+  void ClockInsert(ClockHandle* h);
```

Comment:
Please add a comment. 
I thought some inserts happen in this function, but the function body just SetClockPriority.